api-sample/src/actions/invitations/modifyOrganizationInvitation/queries/updateOrganizationInvitation.js

```diff
@@ -0,0 +1,26 @@
+const {
+  submitQuery,
+  sql,
+  sqlReduce,
+  getInsertId
+} = require("~root/lib/database");
+
+const updateOrganizationInvitation = ({
+  email = null,
+  invitationShortcode,
+  orgId
+}) => {
+  const updates = [];
+  if (email !== null) {
+    updates.push(sql`email = ${email}`);
+  }
+
+  if (updates.length !== 0) {
+    updates.push(sql`updated_at = CURRENT_TIMESTAMP`);
```

Comment:
is this necessary? I can see on the `user_organization_invitations` there is a block like this in the schema


```
updated_at DATETIME DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
```